src/core/controllers/oidc.go

```diff
@@ -175,6 +182,7 @@ func (oc *OIDCController) Onboard() {
 		oc.SendBadRequestError(errors.New("Failed to get OIDC user info from session"))
 		return
 	}
+	defer oc.DelSession(userInfoKey)
```

Comment:
in the previous implementation, line 223 will not delete the session.  But in this PR, it will.

Reply:
Will fix